scripts/BuildAndTest.ps1

```diff
@@ -144,6 +144,12 @@ if (-not $NoRestore) {
 }
 
 if (-not $NoBuild) {
+    & $RepoRoot\src\sarif-pattern-matcher\BuildAndTest.cmd -NoTest
+    & $RepoRoot\src\sarif-pattern-matcher\Src\sarif-sdk\BuildAndTest.cmd -NoTest
```

Comment:
>sarif-sdk [](start = 46, length = 9)

If the SDK is a submodule of Spam, and if Spam actually consumes the SDK binaries (as opposed to consuming a NuGet package), then don't you have to build the SDK first?

And if Spam does _not_ consume SDK binaries, then why do you have to build the SDK at all? For that matter, if that is the case, why are we including the SDK as a submodule at all?

Reply:
I will have to take a look at the spam build and test. i think we can simplify it.
Let's merge for now, so we can do some work, and i will come back to this later (today or in the next few days)

---
In reply to: [534553739](https://github.com/microsoft/sarif-visualstudio-extension/pull/301#discussion_r534553739) [](ancestors = 534553739)

Reply:
Ok, but before merging, please do make that one last change of checking the exit code from each of the two builds separately. Then yes, check whether they are in the right order, and if the SDK build is needed at all.

---
In reply to: [534555694](https://github.com/microsoft/sarif-visualstudio-extension/pull/301#discussion_r534555694) [](ancestors = 534555694,534553739)

Reply:
Oh I see you already did that :-) ok.

---
In reply to: [534556847](https://github.com/microsoft/sarif-visualstudio-extension/pull/301#discussion_r534556847) [](ancestors = 534556847,534555694,534553739)